viz/cmd/uninstall.go

```diff
@@ -43,5 +45,60 @@ func uninstallRunE(ctx context.Context) error {
 		return err
 	}
 
-	return pkgCmd.Uninstall(ctx, k8sAPI, selector)
+	if err := pkgCmd.Uninstall(ctx, k8sAPI, selector); err != nil {
```

Comment:
perhaps a comment here as a reminder that the Uninstall function deletes all cluster scoped resources for the extension, including the extension namespace.